docker-compose.yml

```diff
@@ -2,17 +2,9 @@ version: "3"
 
 services:
   frontend:
-    build: 
-      context: .
+    image: ghcr.io/genomicdatainfrastructure/gdi-userportal-frontend:main
```

Comment:
All these prod deploy related docker changes could have been in a separate PR

Reply:
This `docker-compose.yml` is not prod, I created it to test locally the Dockerfile, to understand if NGINX was getting properly the environment variables.

Anything related to environment deployment is held here: https://github.com/GenomicDataInfrastructure/gdi-userportal-ckan-deployment

Reply:
@admy7 can you please put revert these changes?

Reply:
Done